pytorch_addons/docs/bladedisc_torch_overview.md

```diff
@@ -0,0 +1,112 @@
+## Overview
+
+![bladedisc_torch_overview](./bladedisc_torch_overview.jpg)
+
+The figure shows how a PyTorch nn.Module is compiled via BladeDISC. In short,
+BladeDISC for PyTorch is a compiler to python subset language for deep learning.
+
+A PyTorch nn.Module's forward mehtod is compiled to TorchScript by scripting or
+tracing firstly. Later, some passes are runned to simplify the TorchScript graph
+more closer to SSA(Static Single Assignment) Tensor computations. Then, some
+subgraphs are converted to MHLO. Once BladeDISC get a MHLO module, it will
+optimize and lower the module progressively to different backends.
+
+BladeDISC for PyTorch only supports AOT compilation currently. While JIT's
+roadmap should not differ too much. Please go to
+[Pass Pipeline Walkthrough](../../docs/blade_disc_pass_pipeline_walkthrough.md)
+for more details about the compilation phase after MHLO.
+
+### 1. MLIR/MHLO
+
+MHLO was chosen to be the tensor level "hub" IR between PyTorch and BladeDISC
+compiler world, with the considerations:
+
+- Dynamic shape support
+- Control flow support
+- Available and well established BladeDISC compiler
+- Continuous improvements and updates from the MLIR/XLA community
+
+### 2. Conversion & Lowering
+
+During optimization and compilation, there are many conversions and lowering
```

Comment:
Nit."there are many ... operators"==>"TorchScript will be converted and lowered to MHLO operators with the following work: